### DISCUSSION TITLE
[Feature Request] Webhook event for Organization member role edited

### DISCUSSION THREAD

**Discussion by eshaan-deepsource** [community/community#40531] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

There is no webhook event for when the role of a member in an organization is changed.

GitHub already has a webhook for the `organization` event ([link to docs](https://docs.github.com/en/enterprise-cloud@latest/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#organization)) where a new action `member_role_edited` could be added.

My personal use case is that this should be available to GitHub Apps.

**Comment by ossanna16** [comment]
Hi there @eshaan-deepsource and welcome to our community! Thank you for the feedback!
